The_Lost_Star/common/activities/activity_types/expedition.txt

```diff
@@ -0,0 +1,1000 @@
+﻿activity_expedition = {
+	is_shown = {
+		highest_held_title_tier > tier_barony
+		is_landed = yes
+		is_ai = no
+	}
+	
+	can_start_showing_failures_only = {
+		is_available_adult = yes
+		can_hunt_trigger = yes
```

Comment:
can_hunt_trigger is
`can_hunt_trigger = {
	NOR = {
		has_trait = wounded_2
		has_trait = wounded_3
		has_trait = infirm
		has_trait = maimed
		has_trait = blind
	}
	has_contagious_deadly_disease_trigger = no
	custom_tooltip = {
		text = hunt_cannot_hunt_parameter_tt
		NOT = {
			culture = { has_cultural_parameter = cannot_hunt }
		}
	}
}`

I would argue we should make our own scripted trigger instead
Being blind should be fine
and we don't care about the cultural hunting parameter,
The disease and wound triggers are fine